fetch.bs

```diff
@@ -5343,6 +5377,13 @@ method, must run these steps:
  {{Headers}} object whose <a for=Headers>guard</a> is
  "<code>immutable</code>".
 
+ <li><p>Add the following steps to <var>requestObject</var>'s <a for=Request>signal</a>'s
+ <a>cancelation callbacks</a>:
```

Comment:
No idea if this is an acceptable way to add a sub-algorithm to a list of algorithms.